src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/Runtime/ForwardCancellationTokenToInvocations.Analyzer.cs

```diff
@@ -135,7 +135,7 @@ private void AnalyzeCompilationStart(CompilationStartAnalysisContext context)
 
             // Verify that the current invocation is not passing an explicit token already
             if (AnyArgument(invocation.Arguments,
-                            a => a.Parameter.Type.Equals(cancellationTokenType) && !a.IsImplicit))
+                            a => cancellationTokenType.Equals(a.Parameter?.Type) && !a.IsImplicit))
```

Comment:
@Youssef1313 Is the generated IOperation tree nullable-aware?

Reply:
@mavasani Yes, but for a newer version of Microsoft.CodeAnalysis than what we use here.

That's why #5865 has revealed this bug.

Reply:
For reference, `IArgumentOperation.Parameter` was annotated as nullable in https://github.com/dotnet/roslyn/commit/456829a7f271db03faa2889e2f959b60d8c2ce07.